pkg/server/plugin/upstreamca/awssecret/awslib_fake_test.go

```diff
@@ -0,0 +1,53 @@
+package awssecret
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+)
+
+type fakeSecretsManagerClient struct {
+	secretsmanageriface.SecretsManagerAPI
```

Comment:
don't need to embed this if you define your own interface (see comment in awslib.go)